deploy/vagrant/provision/roles/ufs_swift/files/compile_alluxio.sh

```diff
@@ -0,0 +1,4 @@
+#!/usr/bin/env bash
+
+cd /alluxio
+mvn clean install -Dmaven.javadoc.skip=true -DskipTests -Dlicense.skip=true -Dcheckstyle.skip=true -Dfindbugs.skip=true -Pmesos > mvn.log 2>&1
```

Comment:
answering my own question, I guess you copied that from elsewhere and it is used there because the deploy module supports Mesos